src/components/AmountFilter.jsx

```diff
@@ -10,7 +10,7 @@ export default function AmountFilter(props) {
   const error = max > 0 && max < min;
 
   return (
-    <Grid container spacing={2}>
+    <Grid container justifyContent="space-between" sx={{ maxWidth: 300 }}>
```

Comment:
Interesting! I hadn't noticed how off it'd be on small to medium sized screens. Makes me wonder if we should change the transition breakpoint to [sm](https://mui.com/customization/breakpoints/#default-breakpoints) instead 🤔 .

That being said, maybe add `margin: auto`? I'm good either way.